### ISSUE TITLE
There are global subscriptions

### ISSUE THREAD

**Issue by mitar** [AppWorkshop/meteor-accounts-admin-materializecss#8]
In `startup.js`:

``` javascript
Meteor.startup(function() {
    Meteor.subscribe('roles');
    Deps.autorun(function(e) {
        Meteor.subscribe('filteredUsers', Session.get('userFilter'));
    });
});
```

This is seen as bad practice. It is recommended that you use always a template-level `subscribe`. So that you subscribe only when you need data. Why would all users (even non-admins one, those who should not even get access to this admin interface in my app) open through unnecessary subscriptions?

Use of `Session` (global state) is also discouraged.

**Comment by cunneen** [comment]
Yep, this is inherited from the bootstrap version. Pull requests welcome!

**Comment by cunneen** [comment]
Fixed by @mitar in #10 .
